addon/components/multi-faceted-search/facet.js

```diff
@@ -5,9 +5,11 @@ let Facet = Ember.Object.extend({
   type: "", // "checkbox" or "link"
   terms: [], // each term: { value: "", display: string.optional, count: num.optional, selected: boolean }
   emptyMessage: "Nothing to display",
+  hasError: false,
 
   checkbox: Ember.computed.equal('type', 'checkbox'),
   link: Ember.computed.equal('type', 'link'),
+  datetime: Ember.computed.equal('type', 'datetime'),
```

Comment:
The addon treats them as any other terms in other facets that can be selected / unselected; so no changes were necessary in the `applied-facets`. The engine always keeps them unselected. Since the engine always needs their values, their status of selected / unselected is irrelevant.